src/types/Todo.ts

```diff
@@ -0,0 +1,6 @@
+export interface Todo {
```

Comment:
The indentation does not seem to follow the soft tabs set to 2 spaces rule. Please ensure that the code is using 2 spaces for indentation.